apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -91,6 +132,7 @@ $tokensTable-border-width: 1px;
         padding: f.spacing(e) 0;
         border-radius: 0;
         border-width: 0 0 $tokensTable-border-width 0;
+        background-color: f.$color-background-default;
```

Comment:
Don't think you need this line (looks like removing it in the preview has no effect)